modules/social_features/social_book/social_book.install

```diff
@@ -294,7 +294,7 @@ function social_book_update_11401(): string {
  * Permissions to check:
  * - "translate book node".
  */
-function social_book_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.